### PR TITLE
feat(ui): add selectedNetwork to wallet tooltip

### PR DESCRIPTION
**Pull request by sebastinez**
Hi folks,

This PR adds the functionality requested in #1945.
It uses the `environment` property created by `wallet.ts`
https://github.com/radicle-dev/radicle-upstream/blob/eef980fe3976fa28c98517b6c9a61d7e5fb84d8a/ui/src/wallet.ts#L390

Let me know what you think.
Cheers!

### CODE REVIEW

**Comment by sebastinez** [comment]
I realized that the mainnet case will not be taken into account since There is only the local and ropsten option..
Will see how to address this part of the issue.

P.S. The implementation could eventually be more generic to allow deployments on other test networks for example Kovan, Goerli, etc.

**Comment by rudolfs** [comment]
Hey @sebastinez, this should do fine for now.
We require contributors to both [gpg sign](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits) commits as well as sign-off (`git commit -s`) on our [DCO](https://github.com/radicle-dev/radicle-upstream/blob/master/DCO). Could you do that and force-push the commit after?

**Comment by sebastinez** [comment]
> Hey @sebastinez, this should do fine for now.
> We require contributors to both [gpg sign](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits) commits as well as sign-off (`git commit -s`) on our [DCO](https://github.com/radicle-dev/radicle-upstream/blob/master/DCO). Could you please do that and force-push the commit after?

Thank you @rudolfs

File: ui/DesignSystem/Component/WalletStatusIndicator.svelte [hunk 1]
```diff
@@ -32,7 +32,7 @@
         </SidebarItem>
       </Tooltip>
     {:else}
-      <Tooltip value="Wallet · Connected">
+      <Tooltip value={`Wallet · Connected to ${wallet.environment}`}></Tooltip>
```
Comment (rudolfs) [inline ui/DesignSystem/Component/WalletStatusIndicator.svelte:35]:
```suggestion
      <Tooltip value={`Wallet · Connected to ${wallet.environment}`}>
```

Comment (sebastinez) [reply ui/DesignSystem/Component/WalletStatusIndicator.svelte]: Yes it slipped through in the last commit.. I fixed it in the most recent commit.

**Review by rudolfs — APPROVED** [review]

**Comment by rudolfs** [comment]
Merged via 500d63b013eb4de4b6d675ac1bd86e27aa750345.

**Pull request closed without merging**